File: code-examples/scala/stable-api/basic.scala
import org.mongodb.scala._

// Replace the placeholders in the connection string uri with your credentials
val uri: String = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority"

// Create Settings with API version
val settings: MongoClientSettings = MongoClientSettings.builder()
    .applyConnectionString(new ConnectionString(uri))
    .serverApi(ServerApi.builder().version(ServerApiVersion.V1).build())
    .build()

// Create a client with specified settings
val mongoClient: MongoClient = MongoClient(settings)

File: code-examples/php/stable-api/basic.php
<?php

/* 
 Use PHP MongoDB extension
 Note: Replace the placeholders in the connection string with your credentials 
*/
$uri = 'mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority';

/* 
  Create a client with options to specify Stable API Version 1
  Note: MongoDB PHP Driver doesn't directly provide an option to use MongoDB server API version. Instead API version needs to be passed as an option in the connection string.
  In this case 'serverApi=1' is added to the uri to use stable API Version 1
*/
$uri = $uri . '&serverApi=1';

$client = new MongoDB\Client($uri);

?>
